lib/event.rb

```diff
@@ -0,0 +1,30 @@
+class Event
+
+  attr_reader :name, :crafts, :attendee_names
+
+  def initialize(name, craft, names)
+    @name = name
+    @crafts = [].push(craft).flatten
+    @attendee_names = [].push(names).flatten
```

Comment:
Similar to above, but here we want to set the names list to attendees

```suggestion
    @attendees = names
```

This is because we want an `attr_reader` for attendees. Then we will create a method that will grab the name for us so that we have access to `attendee_names`:

```
def attendee_name
  @attendees.map{|attendee| attendee.name }
end
```